.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: java
 
 script:
-  - ./gradlew build --no-daemon
-  - ./gradlew ktlintCheck --no-daemon
+  - .plugin/gradlew -p ./plugin build --no-daemon
+  - .plugin/gradlew -p ./plugin ktlintCheck --no-daemon
+  - .samples/gradlew -p ./samples :kotlin:ktlintCheck --no-daemon
```

Comment:
I think travis automatically adds `--no-daemon` to all gradle invocations by modifying `~/.gradle/gradle.properties`. But you can keep this if you want.